apps/src/templates/instructions/InlineAudio.jsx

```diff
@@ -87,6 +87,8 @@ const InlineAudio = React.createClass({
     });
 
     audio.addEventListener("error", e => {
+      // e is an instance of a MediaError object
+      trackEvent('InlineAudio', 'error', e.target.error.code);
```

Comment:
Is the error code enough to diagnose what went wrong or should we be tracking additional data?